felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/RiverUtils.kt

```diff
@@ -41,6 +41,11 @@ fun JsonMessage.require(vararg keyAndParserPairs: Pair<IKey, (JsonElement) -> An
     validate(JsonMessage::require, keyStringAndParserPairs)
 }
 
+fun JsonMessage.interestedIn(vararg keyAndParserPairs: Pair<IKey, (JsonElement) -> Any>) {
```

Comment:
Litt kryptisk  - en test ville nok gjort det klarere for meg hva som er tanken bak og hvordan dette skal brukes